src/scry/completion/dependency_graph.cr

```diff
@@ -26,6 +26,10 @@ module Scry::Completion::DependencyGraph
 
   class Graph
     def initialize(@nodes = {} of String => Node)
+      if crystal_path = ENV["CRYSTAL_PATH"]?
+        prelude_path = "#{crystal_path}/prelude.cr"
+        @nodes[prelude_path] = Node.new(prelude_path)
+      end
```

Comment:
@laginha87 Is this like you said?

I trying to test it, but I'm getting `Loading...` forever :sweat_smile: 

![screenshot_20180423_085225](https://user-images.githubusercontent.com/3067335/39130818-a5e153bc-46d3-11e8-8827-5905ec56973d.png)

Reply:
After fixing the `not_nil!` issue I'm can't get stdlib completion:

![screenshot_20180423_090623](https://user-images.githubusercontent.com/3067335/39131643-997e80e8-46d5-11e8-8079-68d36f55e9a4.png)

Reply:
Oh, no `crystal_path` is not right:

```
@nodes: {"/home/main/Projects/console/lib:lib:/usr/lib/crystal/prelude.cr" => Scry::Completion::DependencyGraph::Node
```

Let me fix that :+1:

Reply:
Fixed now!

![screenshot_20180423_103742](https://user-images.githubusercontent.com/3067335/39137070-5af6222e-46e2-11e8-90c1-f07c46cc2c51.png)

I set the `prelude_node`  by default on all files that doen't exist in the `@dependency_graph` :tada: 

So, we can have completion even in Untitled files :sparkles: 

![screenshot_20180423_103945](https://user-images.githubusercontent.com/3067335/39137197-a4e1d432-46e2-11e8-8ec1-52f1bedf4e5d.png)